go-controller/pkg/metrics/master.go

```diff
@@ -567,3 +612,70 @@ func getPodUIDFromPortBinding(row *sbdb.PortBinding) kapimtypes.UID {
 	}
 	return kapimtypes.UID(podUID)
 }
+
+func incrementNbCfg(nbClient libovsdbclient.Client) {
+	nbCfgValue += 1
```

Comment:
Same note as above, I feel like  we should instead be mutating nbCfg +=1 without tracking what we think nbCfg is in memory

Reply:
see above